content/docs/1.5.0/spdk/prerequisites.md

```diff
@@ -5,11 +5,19 @@ weight: 2
 
 ## Prerequisites
 
+Longhorn worker nodes must meet the following requirements:
+
 - x86-64 CPU with SSE4.2 instruction support
-  > NOTICE
+  > **NOTICE**
+  >
   > Currently, V2 Data Engine only supports `x86_64` platform.
 
-- Required linux kernel modules
+- Linux kernel
+
+  5.13 or later is required for NVMe over TCP support
```

Comment:
Why 5.13+? any critical fixes for that or the following versions?

Reply:
there are many bug fixes in older versions and mayastor also encounter issues in old kernel
We can use 5.13 for safety.

https://github.com/openebs/mayastor/releases